apps/07jquery-tic-tac-toe/script.js

```diff
@@ -1,5 +1,68 @@
 'use strict';
 
+// playerTurn variable
+var playerTurn = 'X'
+// variables to store each data-cell value
+var $cell1 = $('[data-cell = "0"]')
+var $cell2 = $('[data-cell = "1"]')
+var $cell3 = $('[data-cell = "2"]')
+var $cell4 = $('[data-cell = "3"]')
+var $cell5 = $('[data-cell = "4"]')
+var $cell6 = $('[data-cell = "5"]')
+var $cell7 = $('[data-cell = "6"]')
+var $cell8 = $('[data-cell = "7"]')
+var $cell9 = $('[data-cell = "8"]')
+
 $(document).on('ready', function() {
-  // Put app logic in here
+  $('[data-cell]').click(function(){
+    $(this).text(playerTurn);
+    checkForWin();
+    playerTurn = (playerTurn === 'X') ? 'O' : 'X';
+  });
+  clearBoard();
+  
 });
+
+// function to check for win
+function checkForWin() {
+  // if all win functions evaluate to true, announce winner
+  if (horizontalWin() === true || verticalWin() === true || diagonalWin() === true) {
```

Comment:
You can pass in these 3 function directly and your if statement will automatically check for a falsey or truthy value..

ex: `if(returnTrue() || returnFalse()) { return "one of these functions returned true!" }`